cpu/samd5x/periph/can.c

```diff
@@ -427,9 +436,14 @@ static int _init(candev_t *candev)
     if (IS_ACTIVE(ENABLE_DEBUG)) {
         _dump_msg_ram_section(dev);
     }
-    /* Disable automatic retransmission by default */
-    /* This can be added as a configuration parameter for the CAN controller */
-    dev->conf->can->CCCR.reg |= CAN_CCCR_DAR;
+
+    if (dev->conf->disable_automatic_retransmission) {
+        dev->conf->can->CCCR.reg |= CAN_CCCR_DAR;
```

Comment:
avoid single bit writes

prepare the configuration word and write that

Reply:
Out of scope. The whole initialization is a complete mess; but fixing this is not the goal here.